plugins/toolbar/prism-toolbar.js

```diff
@@ -0,0 +1,82 @@
+(function(){
+	if (typeof self === 'undefined' || !self.Prism || !self.document) {
+		return;
+	}
+
+	var callbacks = [];
+
+	Prism.plugins.toolbar = {};
+
+	/**
+	 * Register a button callback with the toolbar.
+	 *
+	 * @param {Object|Function} opts
+	 */
+	Prism.plugins.toolbar.registerButton = function (opts) {
+		var callback;
+
+		if (typeof opts === 'function') {
+			callback = opts;
+		} else {
+			callback = function (env) {
+				var element;
+
+				if (typeof opts.onClick === 'function') {
+					element = document.createElement('a');
+					element.addEventListener('click', function () {
+						opts.onClick(env);
```

Comment:
I don't know how important it is, but I didn't want to change the `this` value of the passed in callback in case the developer is relying on it.